2022/Day9/RFC.md

```diff
@@ -21,6 +21,47 @@ You just need to work out where the tail goes as the head follows a series of mo
 both start at the same position, overlapping.
 
 ## Glossary:
+
+## Part 2:
+### Goal:
+Rather than two knots, you now must simulate a rope consisting of ten knots. One knot is still the head of the rope and 
+moves according to the series of motions. Each knot further down the rope follows the knot in front of it using the same 
+rules as before.
+
+Simulate your complete series of motions on a larger rope with ten knots. How many positions does the tail of the rope 
+visit at least once?
+
+### Out of Scope:
+What problems are you not trying to solve?
+
+### Design:
+
+#### Parsing:
+No additional parsing needed
+
+#### Solution:
+We will start by creating a class called `Rope` that will keep track of `Rope_Segments` in the rope via an array called,
+`rope_segments`. It will have the following methods:`get_head`, `get_tail`, `move_segments`, and `add_segment_to_end`.
+`get_head` and `get_tail` will return the first/last `Rope_Segments` in the `rope_segments`. 
```

Comment:
This sounds like a linked list, rather than an array.

Reply:
you're right - it is a linked list rather than an array